pkg/nl/layer3.go

```diff
@@ -164,3 +165,33 @@ func (*NetlinkManager) EnsureBPFProgram(info VRFInformation) error {
 
 	return nil
 }
+
+func (info VRFInformation) linkMTU() int {
+	if info.MTU == 0 {
+		return defaultMtu
```

Comment:
Could also be done using a defaulting webhook so it's also represented in the CRD. Not mandatory though, it might be excessive to have one just for that.

Reply:
it's already set as a default in the CRD and that apparently sets it in the cluster as well (without using a webhook)